test/test.cc

```diff
@@ -4258,7 +4258,7 @@ TEST_F(ServerTest, PutLargeFileWithGzip2) {
   ASSERT_TRUE(res);
   EXPECT_EQ(StatusCode::OK_200, res->status);
   EXPECT_EQ(LARGE_DATA, res->body);
-  EXPECT_EQ(101942u, res.get_request_header_value_u64("Content-Length"));
+  EXPECT_TRUE(res.get_request_header_value_u64("Content-Length") < (10 * 1024 * 1024));
```

Comment:
Instead of `EXPECT_TRUE`, it'd be better to use [`EXPECT_LT`](https://google.github.io/googletest/reference/assertions.html#EXPECT_LT)

Reply:
@sebastianas could you also put some comment before `EXPECT_LT`, so that it clearly explains why this check doesn't use `EXPECT_EQ`? Thanks!

Reply:
both changes have been made.